client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -31,6 +49,48 @@ export default ({ siteId }) => {
 
   const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
 
+  const forceReload = async () => {
```

Comment:
is this code taken from the useEffect on line 94? I think we would be better off encapsulating it.

Reply:
I know this is an anti pattern, but we're kinda forced into it. We need to be able to reload the form after we submit. 
If we just have it declared within the useEffect, then we can't call it elsewhere. If we declare it outside, it becomes a dependancy of the useEffect and breaks. 
I don't know if there is a more elegant way to handle this issue. 

We use this pattern exactly in the participants table.

Reply:
Ah, fair enough! I forgot about the whole dependency thing.